src/com/company/Transport.java

```diff
@@ -0,0 +1,28 @@
+package com.company;
+
+abstract class Transport implements ServiceStationInterface {
+    private String modelName;
+    private  int wheelsCount;
+
+    public Transport(String modelName,int wheelsCount) {
+        this.modelName = modelName;
+        this.wheelsCount = wheelsCount;
+    }
+
+    public String getModelName() {
+        return modelName;
+    }
+
+    public int getWheelsCount() {
+       return wheelsCount; }
+
+
+    public void updateTyre() {
+                 System.out.println("Меняем покрышку");
+             }
```

Comment:
```suggestion
    public void updateTyre() {
                 System.out.println("Меняем покрышку");
    }
```
 рекомендую использовать ctrl+alt+L для автоформатирования кода